outpost/pkg/ldap/utils.go

```diff
@@ -94,14 +94,12 @@ func (pi *ProviderInstance) APIGroupToLDAPGroup(g api.Group) LDAPGroup {
 }
 
 func (pi *ProviderInstance) APIUserToLDAPGroup(u api.User) LDAPGroup {
-	dn := fmt.Sprintf("cn=%s,%s", u.Username, pi.UserDN)
-
 	return LDAPGroup{
-		dn:				dn,
+		dn:				fmt.Sprintf("cn=%s,%s", u.Username, pi.GroupDN),
```

Comment:
Any reason to not use `pi.GetUserDN` here too?

Reply:
Well, since it's a technically a group we shouldn't be using the users dn for the record, right?
However, I should probably rewrite the GetGroupDN function to accept the string name (like GetUserDN) and use that instead of Sprintf

Reply:
Yeah I just noticed that I misread, my bad

Reply:
Refactored `pi.GetGroupDN` is now commited as well